src/main/java/com/nscharrenberg/um/multiagentsurveillance/headless/models/Player/Intruder.java

```diff
@@ -1,20 +1,33 @@
 package com.nscharrenberg.um.multiagentsurveillance.headless.models.Player;
 
+import com.nscharrenberg.um.multiagentsurveillance.headless.Factory;
+import com.nscharrenberg.um.multiagentsurveillance.headless.models.Angle.AdvancedAngle;
 import com.nscharrenberg.um.multiagentsurveillance.headless.models.Angle.Angle;
 import com.nscharrenberg.um.multiagentsurveillance.headless.models.Map.Tile;
+import com.nscharrenberg.um.multiagentsurveillance.headless.models.Map.TileArea;
+import com.nscharrenberg.um.multiagentsurveillance.headless.utils.files.MapImporter;
+import com.nscharrenberg.um.multiagentsurveillance.headless.utils.TargetDirection;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
 import java.util.Objects;
 
 public class Intruder extends Player {
     private double sprintSpeed;
     private boolean isSprinting = false;
+    private Angle targetDirection;
```

Comment:
What's the difference between `targetDirection` and `targetDirectionAdvancedAngle`?

Reply:
Sorry for the delayed reply. targetDirection only gives LEFT RIGHT UP DOWN  targetDirectionAdvancedAngle gives also TOP_LEFT, TOP_RIGHT, BOTTOM_LEFT, BOTTOM_RIGHT. Do you want me to delete one of them?